README.md

```diff
@@ -1 +1,44 @@
-# school-library-ruby
\ No newline at end of file
+# OOP School Library 
+
+> This project implements classes and objects in Ruby, Implementing encapsulation and inheritance features
+
+## Getting Started 
+
+To run this project on your machine, clone the repo 
+
+``` git clone https://github.com/mutinhiri/school-library-ruby```
+
+and open the directory ```school library ruby``
```

Comment:
- [ ] the code block in this line is not working due to a tilde sign which is missing. 
- [ ] it would be nice if the potential users of your application can just copy and paste the code blocks and that by doing this your application will run, so instead of `school library ruby` make it look like `cd school-library-ruby`